### ISSUE TITLE
Uncaught exception when setting up openfl (with haxe installed via homebrew)

### ISSUE THREAD

**Issue by jordgubben** [openfl/openfl#764]
Attempting to setup openfl on OS X 10.10 (with haxe installed by homebrew) causes this exception:

    DeLorean:~ jakob$ haxelib run openfl setup
    Called from ? line 1
    Called from RunScript.hx line 40
    Called from helpers/PathHelper.hx line 197
    Uncaught exception - Invalid field access : split

From PathHelper.hx (in lime):

```haxe
try {
	var cacheDryRun = ProcessHelper.dryRun;
	ProcessHelper.dryRun = false;
	output = ProcessHelper.runProcess (Sys.getEnv ("HAXEPATH"), "haxelib", [ "path", name ], true, true, true);
	ProcessHelper.dryRun = cacheDryRun;
} catch (e:Dynamic) { }

LogHelper.verbose = cache;
			
var lines = output.split ("\n");
var result = "";
	
```
Line 197 is a blank line, but the above lines are what surrounds it (blank lines have been removed for compactness). My HAXEPATH is not set, so that might be a part of it. If something goes wrong in ProcessHandler.runProcess() and it returns null, then that would explain why the *output*  string a few lines down is missing the split() method. But this is as far as my current haxe-skills take me.

### Environment
    DeLorean:~ jakob$ haxe -version
    3.2.0
    DeLorean:~ jakob$ haxelib list
    lime: [2.5.2]
    openfl: [3.2.2]

## Saving clause(s)
I'm also having possibly related problems setting up lime. I've reported a [separate issue](https://github.com/openfl/lime/issues/519)  about that since it might also be completely unrelated.

I must admit I'm to new to the haxe-ecosystem to tell if this is a problem with homebrew, haxelib or openfl. But since I've not encountered any other haxe-lib with this priblem (yet). Thus I'm thought I'd start here and then work my way "uppstream" if this is not a lime issue.

**Comment by Bergerton** [comment]
Not sure if you found your fix for this or not, but I thought I'd leave my solution for anybody else who stumbles this way.

I installed haxe through the standalone installer, uninstalled my brew version of haxe, and everything worked for my openfl setup.

**Comment by jgranick** [comment]
I think the homebrew install might break `haxelib path lime`, which might be the cause of this error.

**Comment by Bergerton** [comment]
Wouldn't surprise me, using homebrew seems to have set me back more than anything last night.
(I had to revert to updating flambe manually as well)

**Comment by jeffreytgilbert** [comment]
I found with my instance that I had some lingering bins installed in /usr/bin and as soon as I removed them, it instantly started working. I did also run the selfupdate, but it did not fix the problem. It just showed me what was wrong. haxelib was reporting back with the wrong version after the self update, and when it ran I got a permissions denied error, which was odd at the time, but now makes total sense. Folders were missing, libs had the wrong versions, etc.
